the-name-card/src/components/Section/Section.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react';
+import './Section.scss';
+
+interface SectionProps {
+	children: ReactNode;
```

Comment:
this is 1 valid way to do this

here are some other ways
https://stackoverflow.com/questions/71788254/react-18-typescript-children-fc
https://stackoverflow.com/questions/71788254/react-18-typescript-children-fc/71800185#71800185